sdk/textanalytics/ai-text-analytics/review/ai-text-analytics.api.md

```diff
@@ -193,6 +193,9 @@ export interface RecognizeCategorizedEntitiesSuccessResult extends TextAnalytics
     readonly entities: CategorizedEntity[];
 }
 
+// @public (undocumented)
```

Comment:
Can we have the docs added for the new type as well?

Reply:
Done.